src/anupSJun20/ArmstrongNumber.java

```diff
@@ -0,0 +1,28 @@
+package anupSJun20;
+/*Program 4: WAP to check if a number is Armstrong or not.
+Example:-
+Input:- 153
+Output:-153
+hint:- (1*1*1)+(5*5*5)+(3*3*3)
+1+125+27
+153*/
+
+public class ArmstrongNumber {
+	static void findANumberArmstrong(int num) {
+		int input = num;
+		int newNum = 0;
+		while(num>0) {
+			newNum = newNum+ (num%10)*(num%10)*(num%10);
```

Comment:
**Suggestion:**
You can take a look at power function - Math.pow(x,y)
which will help you achieve same result for multiplication.
i.e. Math.pow((number % 10), 3))

Reply:
Thanks Aditi. Changes incorporated. Didn't think of this function.